browser/binance/android/binance_native_worker.cc

```diff
@@ -104,7 +104,7 @@ BinanceNativeWorker::GetOAuthClientUrl(JNIEnv* env) {
 
 void BinanceNativeWorker::GetAccessToken(JNIEnv* env) {
   if (binance_service_) {
-    binance_service_->GetAccessToken(base::Bind(
+    binance_service_->GetAccessToken(base::BindRepeating(
```

Comment:
I think `BindOnce` should be used here and all others in this file.
It's one-shot callback.

Reply:
Oops! I'm not sure how I overlooked those methods which all take a OnceCallback callback, since I've been carefully checking things in all places.. but it seems I slipped here, thanks for the catch!

@simonhong Fixed now in the PR, PTAL thanks!